libraries/botframework-streaming/src/webSocket/index-browser.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * @module botframework-streaming
+ */
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+export * from './browserWebSocket';

```

Comment:
nit: `export *` is potentially dangerous and I'd prefer being specific on which objects are being exported, especially given the fragility of our browser support without CI based testing.